libvast/vast/plugin.hpp

```diff
@@ -44,6 +44,14 @@ extern "C" struct plugin_version {
   uint16_t minor;
   uint16_t patch;
   uint16_t tweak;
+
+  /// An additional field containing the version of the libvast that is linked
+  /// against. Should compare equal to VAST_VERSION.
+  const char* libvast_version;
```

Comment:
Why are the preprocessor variables called `VAST_` but the ones here `libvast_*`?

Reply:
In terms of the plugin, it is libvast that is relevant for us. Within libvast, all macros are prefix with `VAST_`.